### PR TITLE
migration: move events to wholesale container

### PR DESCRIPTION
**Pull request by JesperGomes**
## References

* #508

### CODE REVIEW

File: source/databricks/package/datamigration/migration_scripts/202212051200_Move_Events_To_Wholesale_Container.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        args.storage_account_url,
+        args.storage_account_key,
+        source_container,
+        events_checkpoint_source_directory,
+        destination_container,
+        events_checkpoint_destination_directory,
+    )
+
+
+def move_directory(
```
Comment (BjarkeMeier) [inline source/databricks/package/datamigration/migration_scripts/202212051200_Move_Events_To_Wholesale_Container.py:49]: Looks like something we would love to reuse. Can we move it to somewhere more general?

File: source/databricks/package/infrastructure.py [hunk 2]
```diff
@@ -27,11 +26,11 @@ def get_storage_account_url(storage_account_name: str) -> str:
 
 
 def get_integration_events_path(storage_account_name: str) -> str:
-    return f"abfss://{INTEGRATION_EVENTS_CONTAINER_NAME}@{storage_account_name}.dfs.core.windows.net/{EVENTS_FOLDER_NAME}"
+    return f"abfss://{WHOLESALE_CONTAINER_NAME}@{storage_account_name}.dfs.core.windows.net/{EVENTS_FOLDER_NAME}"
 
 
 def get_integration_events_checkpoint_path(storage_account_name: str) -> str:
-    return f"abfss://{INTEGRATION_EVENTS_CONTAINER_NAME}@{storage_account_name}.dfs.core.windows.net/{EVENTS_CHECKPOINT_FOLDER_NAME}"
+    return f"abfss://{WHOLESALE_CONTAINER_NAME}@{storage_account_name}.dfs.core.windows.net/{EVENTS_CHECKPOINT_FOLDER_NAME}"
```
Comment (BjarkeMeier) [inline source/databricks/package/infrastructure.py:33]: Is it time to stop repeating the whole string (`"abfss://{WHOLESALE_CONTAINER_NAME}@{storage_account_name}.dfs.core.windows.net/`)?

**Review by BjarkeMeier — DISMISSED** [review]

File: source/databricks/package/infrastructure.py (hunk 2 above)
Comment (JesperGomes) [reply source/databricks/package/infrastructure.py]: Done

File: source/databricks/package/datamigration/migration_scripts/202212051200_Move_Events_To_Wholesale_Container.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+        source_container,
+        events_source_directory,
+        destination_container,
+        events_destination_directory,
+    )
+
+    # move 'events-checkpoint'
+    events_checkpoint_source_directory = "events-checkpoint"
+    events_checkpoint_destination_directory = events_checkpoint_source_directory
+    move_directory(
```
[the remark this answers is not in the document]
Comment (JesperGomes) [reply source/databricks/package/datamigration/migration_scripts/202212051200_Move_Events_To_Wholesale_Container.py:38]: yes. I would like to not do it now, though.

File: source/databricks/package/datamigration/migration_scripts/202212051200_Move_Events_To_Wholesale_Container.py (hunk 1 above)
Comment (JesperGomes) [reply source/databricks/package/datamigration/migration_scripts/202212051200_Move_Events_To_Wholesale_Container.py:48]: yes. I would like to not do it now, though.

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**